dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/oracle/OracleStatsQuery.java

```diff
@@ -39,9 +41,10 @@ public abstract class OracleStatsQuery {
   @Nonnull
   abstract StatsSource statsSource();
 
-  static OracleStatsQuery create(String name, StatsSource statsSource) throws IOException {
+  static OracleStatsQuery create(String name, StatsSource statsSource, int days)
```

Comment:
I would suggest keeping the name of the accessor and the parameter the same, e.g. `queriedDays`.

Reply:
Done